src/main/java/CalculatorTest.java

```diff
@@ -0,0 +1,75 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class CalculatorTest {
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+    @ParameterizedTest
+    @NullSource
+    void inputNullTest(String input) {
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            Calculator calculator = new Calculator(input);
+        });
+    }
+    @ParameterizedTest
+    @ValueSource(strings = {"", "  "})
+    void inputEmptyTest(String input) {
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            Calculator calculator = new Calculator(input);
+        });
+    }
```

Comment:
`@NullAndEmptySource`를 활용하시면 반복을 줄일 수 있을거 같아요:)
```suggestion
    @ParameterizedTest
    @NullAndEmptySource
    void inputNullTest(String input) {
        assertThatIllegalArgumentException().isThrownBy(() -> new Calculator(input));
    }
```